hack/functest.sh

```diff
@@ -15,6 +15,7 @@ echo "Running e2e test suite"
 export KUBECONFIG=$(./kubevirtci kubeconfig)
 export NODE_VM_IMAGE_TEMPLATE=quay.io/kubevirtci/fedora-kubeadm:35
 export IMAGE_REPO=k8s.gcr.io
+export TENANT_CLUSTER_KUBERNETES_VERSION=v1.21.0
```

Comment:
now that we're passing cli args to the e2e tets, where do you think the line is for what belongs in an environment variables and what belongs as cli args? Or should we begin transitioning everything that's potentially needed by the test suite as cli args?

Reply:
good question! yes, I'm also puzzled over this, essentially we have two different ways of passing test args ... actually, I like passing all args as cli args, seems cleaner to me since these are all attributes of a test run, but requires a bit more of boiler plate code. here, I added it as env var because version of Kubernetes is related to image we use for tenant cluster, and we already pass the image via env var. but, ideally, maybe we should have only one way of passing args, whether env var or cli arg. using env vars for all args will also work, as long as we have a single consistent method.

Reply:
in any case, let's think about it, and perhaps plan an enhancement for a future PR?

Reply:
yeah, makes sense. I think what you have now looks good